commit/core.pyx

```diff
@@ -890,16 +923,17 @@ cdef class Evaluation :
             print( '\t\t- Coefficients txt files... ', end='' )
             sys.stdout.flush()
             if len(self.KERNELS['wmr']) > 0 :
+                kept = np.fromfile( pjoin(self.get_config('TRACKING_path'),'dictionary_TRK_kept.dict'), dtype=np.bool_)
+                xic = np.zeros( kept.shape )
                 offset = nF * self.KERNELS['wmr'].shape[0]
-                np.savetxt(pjoin(RESULTS_path,'xic.txt'), x[:offset], fmt='%12.5e')
+                xic[kept] = x[:offset]
```

Comment:
This may be not compatible with `CylinderZeppelinBall` model

Reply:
Should be fixed now